Modules/Loadable/SubjectHierarchy/Widgets/qSlicerSubjectHierarchyPluginLogic.cxx

```diff
@@ -64,11 +75,15 @@ qSlicerSubjectHierarchyPluginLogicPrivate::qSlicerSubjectHierarchyPluginLogicPri
   // Register vtkIdType for use in python for subject hierarchy item IDs
   qRegisterMetaType<vtkIdType>("vtkIdType");
   //qRegisterMetaType<QList<vtkIdType> >("QList<vtkIdType>"); //TODO: Allows returning it but cannot be used (e.g. pluginHandler->currentItems())
+
+  this->ViewMenu = new QMenu(); //TODO: Parent?
 }
 
 //-----------------------------------------------------------------------------
 qSlicerSubjectHierarchyPluginLogicPrivate::~qSlicerSubjectHierarchyPluginLogicPrivate()
-= default;
+{
+  this->ViewMenu->deleteLater(); //TODO: OK?
```

Comment:
Looks good to me. Maybe set this->ViewMenu to nullptr at the end to make the behavior in case of an error (object double-delete, use-after-delete) more predictable.